src/rgw/rgw_sync_module_aws.cc

```diff
@@ -1598,12 +1598,6 @@ class RGWAWSHandleRemoteObjCBCR: public RGWStatRemoteObjCBCR {
           src_pg_ver = 0; /* all or nothing */
         }
       }
-      ldout(sync_env->cct, 0) << "AWS: download begin: z=" << sync_env->source_zone
-                              << " b=" << bucket_info.bucket << " k=" << key << " size=" << size
-                              << " mtime=" << mtime << " etag=" << etag
-                              << " zone_short_id=" << src_zone_short_id << " pg_ver=" << src_pg_ver
-                              << " attrs=" << attrs
-                              << dendl;
```

Comment:
i agree that we should drop the attrs, but i feel like the rest is still useful to log - maybe at a higher log level like 4?

Reply:
done